### PR TITLE
update documentation for OZ and ON configuration

### LINKED ISSUE #1691: Verzoek tot bijwerken handleiding
### Thema / Theme

Admin

### Omschrijving / Description

Het stappenplan om Open Zaak op de juiste manier te verbinden met Open Notificaties lijkt niet volledig. 

Het gaat om deze pagina: https://github.com/open-zaak/open-zaak/blob/60698cc05503342ed9a54f8c38e9a7fe04255670/docs/installation/config/openzaak_config.rst

Als concreet voorbeeld: er staan twee keer passages als "Client ID: The same Client ID as given in Open Zaak step 3c", die verwijst naar een stap die ontbreekt in de documentatie.

Als inconcreet voorbeeld: na veel trial and error lukt de configuratie meestal, maar achteraf weten we niet waarom. Graag willen we dat deze stappen reproduceerbaar zijn. 

### Toegevoegde waarde / Added value

Drempelverlagende toevoeging voor effectieve ingebruikname van Open Zaak

### Aanvullende opmerkingen / Additional context

Voor de gemeente Rotterdam moeten we nog twee Open Zaak omgevingen configureren (acc en prod), dus we kunnen die als toetsingscasus gebruiken om de handleiding aan te scherpen en te toetsen. We helpen graag mee.

### PR DESCRIPTION
**Pull request by annashamray**
Fixes #1691

### CODE REVIEW

File: docs/installation/config/openzaak_config.rst [hunk 1]
```diff
@@ -253,10 +213,93 @@ Register the required channels:
...
+        ``https://notificaties.gemeente.local/api/v1/schema/openapi.yaml``
+      - **User ID**: *Same as the Client ID*
+      - **User representation**: *For example:* ``Open Notificaties``
+
+   d. Click **Opslaan**.
+
+9. Configure the Open Notificaties Notificatiescomponent API endpoint (so Open Notificaties
+   receives changes made in the authorisation component of Open Zaak ):
+
+   a. After the previous step 8d you should be on **Configuratie > Notificatiescomponentconfiguratie** page.
```
Comment (Coperh) [inline docs/installation/config/openzaak_config.rst:258]: Something I notice is that Github converts the `d` to `iv`.  I wonder if readthedocs does the same

**Review by Coperh — APPROVED** [review]
Seems good. Cannot find any mistakes and the instructions seem clear

File: docs/installation/config/openzaak_config.rst (hunk 1 above)
Comment (Coperh) [reply docs/installation/config/openzaak_config.rst:258]: As in 8.d is 8.iv

**Review by bart-maykin — APPROVED** [review]

File: docs/installation/config/openzaak_config.rst (hunk 1 above)
Comment (annashamray) [reply docs/installation/config/openzaak_config.rst:258]: I've checked it in RTD and it looks fine:
image

You can check the RTD build locally:
```
cd docs
make html
```

And then open `_build/index.html` in the browser

**Pull request merged**